docs/install/install.md

```diff
@@ -0,0 +1,29 @@
+
+
+
+
+# Installing Half
+
+The library in its most recent version can be obtained from here, see the [Release Notes](changelog.html) for further information:
```

Comment:
don't use "here" - it should be a link.
However, is the Changelog the right place